Source/swiftlint/Commands/LintCommand.swift

```diff
@@ -137,6 +150,8 @@ struct LintOptions: OptionsProtocol {
             <*> mode <| configOption
             <*> mode <| Option(key: "strict", defaultValue: false,
                                usage: "fail on warnings")
+            <*> mode <| Option(key: "lenient", defaultValue: false,
+                               usage: "succeed on errors")
```

Comment:
`"downgrades serious violations to warnings, and any warning threshold is disabled"`